README.md

```
@@ -1,63 +1,63 @@
 
 To see how to Contribute, visit [Getting Started](./DEVELOPERS.md)
 
 ## Community & Support
 
-- [Community Forum](https://github.com/supabase/supabase/discussions). Best for: help with building, discussion about database best practices.
-- [GitHub Issues](https://github.com/supabase/supabase/issues). Best for: bugs and errors you encounter using Supabase.
-- [Email Support](https://supabase.com/docs/support#business-support). Best for: problems with your database or infrastructure.
-- [Discord](https://discord.supabase.com). Best for: sharing your applications and hanging out with the community.
+- [Community Forum](https://github.com/aripitek/supabase/supabase/discussions). Best for: help with building, discussion about database best practices.
+- [GitHub Issues](https://github.com/aripitek/supabase/supabase/issues). Best for: bugs and errors you encounter using Supabase.
+- [Email Support]((https://supabase.com/aripitek/docs/support#business-suppor(. Best for: problems with your database or infrastructure.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)